.github/workflows/ci-prod.yml

```diff
@@ -88,28 +88,47 @@ jobs:
     - name: Upload MSIX package
       uses: actions/upload-artifact@v4
       with:
-        name: MSIX Package ${{ matrix.platform }}
+        name: MSIX Package x64
         path: Packages\*.msix
 
-    - name: Download all MSIX packages
-      uses: actions/download-artifact@v3
+    - name: Upload MSIX package
+      uses: actions/upload-artifact@v4
       with:
-        path: downloaded-packages
-        
-    - name: Get Project version from Package.appxmanifest
-      id: project-version
-      shell: pwsh
-      run: |
-        [xml]$manifest = Get-Content -Path './Package.appxmanifest' # Make sure to update the path for this file
-        $version = $manifest.Package.Identity.Version
-        echo "::set-output name=current-version::$version"
+        name: MSIX Package x86
+        path: Packages\*.msix
```

Comment:
this is running duplicate code. please revert back to using matrix.platform and delete the x86 step.

    - name: Upload MSIX package
      uses: actions/upload-artifact@v4
      with:
        name: MSIX Package ${{ matrix.platform }}
        path: Packages\*.msix